src/trunk.c

```diff
@@ -719,7 +723,9 @@ struct trunk_compact_bundle_req {
    uint64                output_pivot_kv_byte_count[TRUNK_MAX_PIVOTS];
    uint64                tuples_reclaimed;
    uint64                kv_bytes_reclaimed;
-   uint32               *fp_arr;
+   fp_hdr                breq_fingerprint;
+   uint64                num_tuples;
```

Comment:
NOTE to me: Revisit field. fp_hdr has ntuples inside it already. This field was probably added in early rounds before `fp_hdr` was formalized.